source/_docs/pantheon-yml.md

```diff
@@ -1,12 +1,53 @@
 ---
-title: The pantheon.yml Configuration File
-description: Set site configuration in a pantheon.yml file stored in the root of their code repository.  
+title: Pantheon YAML Configuration Files
+description: Learn how to use a site-local configuration file to manage advanced settings at the site level and how to set default configurations across all sites running a Custom Upstream.
 tags: [pantheonyml, infrastructure]
 categories: []
 ---
+Hook into platform workflows and advanced site configurations using Pantheon's YAML configuration files, `pantheon.upstream.yml` and `pantheon.yml`. Both configuration files utilize the same set of properties that define how your Pantheon environments will behave.
```

Comment:
This information *is* present below, but I think it's important to have it right here in the 1st or 2nd sentence so folks are clear on the context up-front.